markdown/articles/2020/03/22/DDFAQs.md

```diff
@@ -0,0 +1,600 @@
+
+**DDEFT
+
+DDEFT tends to skimp a little bit on permission. Normally only 6-7 pieces of permission main. These
+also should always be proactive pieces. If {{Experimental Frenzy}} is on the field and you hit
+a counterspell on top you are then stuck with it present on the top of your deck or, in hand
+where it cannot be cast at all. {{Thoughtseize}} and {{Duress}} are the favourites over other
+splash options like {{Veil of Summer}} as trying to balance mana when playing off the top of your
+deck can really put a strain on having the right colours and you always want {{B}} in adbundance.
+
+**Meandeck
+
+Meandeck probably has the most dedicated protection slots and the widest range of available
+protection/permission to it. It tends to always run at minimum 4 {{Force of Will}} and that is
+supported by another 7-8 protection slots. These can be, depending on the splash colours run,
+any ratio of discard, veils, chant effects, soft counters or similar and sometimes include
+pieces of maindeck removal like {{Abrupt Decay}} too.
+
+
+### 2.5 Combo Pieces
+
+Although the deck is normally a "one card combo" there are certain other pieces needed to make
+it work it the most efficient way possible. These are often cards that may have limited
+usefulness outside of winning the game. Normally decks have 1-2 raw card advantage engine
+pieces {{Ideas Unbound}}, {{Act on Impulse}} or {{Predict}} to enable drawing into the Doomsday
+pile. In additon some number of cards (0-4) to facilitate getting through the piles easier like
+{{Echo of Eons}}, {{Street Wraith}} or {{Edge of Autumn}} are run to make it more mana efficient
+to get the win.
+
+{{Lion's Eye Diamond}} falls into this category. Although it can be used to provide a mana boost
+when digging into a Doomsday pile or used to cast something grabbed with Burning Wish, it generally
+just sits pretty doing nothing prior to that. It is however one of the most powerful tools
+when performing the combo and thus should always be run, albeit to varying degrees.
+
+Finally some sort of win condition is required. This can be in the form of {{Tendrils of Agony}},
+{{Thassa's Oracle}} or even {{Collective Brutatlity}}. {{Burning Wish}} can also take this spot
+as it can tutor up a Tendrils if needed. You want to run as few "dead" cards as possibly and 
+generally the win condition is counted as such so you only want 1-2 of them.
+
+**DDFT
+
+DDFT tends to run 1 card advantage spell and this is normally {{Act on Impulse}} or {{Cruel Bargain}}.
+It can also then run either 0-2 Cycle effects and 0-3 Echo of Eons with a potential 4th as a wish
+target in the sideboard. Generally the win conditions are 0-1 Tendrils main (0 if the Tendrils is
+wishable from the side) and 1 Oracle main. Tendrils is retained as it takes the requirement to have
+Doomsday resolve out by giving the option to win from just a natural storm-style kill. DDFT
+lists should run 4 Lion's Eye Diamonds as they have a lot of utility.
```

Comment:
I'd include more about LED. They're free storm and Make loads of mana to help cast spells you wish for, Like EtW or Echo, as well as being important in piiles

Reply:
Partially already included in 316-319 but added a little more on it.,